docker-run.sh

```diff
@@ -6,13 +6,12 @@
 #
 
 source benchmark-scripts/get-gpu-info.sh
+WORKLOAD_SCRIPT="docker-run-dlstreamer.sh"
```

Comment:
please add a comment that if --workload is not provided this env is the default value will be used.